src/proj.rs

```diff
@@ -121,8 +121,10 @@ pub fn enable_network(enable: bool) -> Result<(), ProjError> {
     }
     let enable = if enable { 1 } else { 0 };
     let dctx: *mut PJ_CONTEXT = ptr::null_mut();
-    unsafe { proj_context_set_enable_network(dctx, enable) };
-    Ok(())
+    match unsafe { proj_context_set_enable_network(dctx, enable) } {
+        1 => Ok(1),
+        _ => Err(ProjError::Network),
```

Comment:
My understanding is that with this change we'll return an error when calling:

```
enabled_network(enable: false)
```

from `proj_context_set_enable_network`:
```
...
    return ctx->networking.enabled &&
           ctx->networking.open != NS_PROJ::no_op_network_open;
```

But was that the intention?

Reply:
Well, this is embarrassing. I'll revisit tomorrow post-caffeine.

Reply:
I think this is now correct…